public/stylesheets/product_listing.css

```diff
@@ -0,0 +1,195 @@
+.prod_list {
+  display: flex; }
+  .prod_list .navigation {
+    background-color: #eaeaea;
+    padding-left: 10px;
+    max-width: 300px;
+    width: 100%;
+    margin-bottom: 0; }
+    .prod_list .navigation > ul {
+      display: block;
+      padding: 0; }
+      .prod_list .navigation > ul li {
+        display: block;
+        border-bottom: 1px solid #d3d3d3;
+        padding: 5px 0px 10px 0px; }
+        .prod_list .navigation > ul li a {
+          color: #747474;
+          padding: 13px 0 13px 25px;
+          font-weight: 400;
+          text-decoration: none;
+          font-size: 16px; }
+        .prod_list .navigation > ul li::hover {
+          color: #000; }
+    .prod_list .navigation input,
+    .prod_list .navigation label {
+      display: none; }
+  .prod_list .product_list_menu {
+    width: calc(100% - 215px);
+    margin-top: 10px; }
+    .prod_list .product_list_menu .prod_desc p {
+      background-color: #f0f0f0;
+      padding: 10px 8px 8px;
+      margin: 15px 0;
+      font-size: 15px;
+      height: 7.2em;
+      overflow: hidden; }
+    .prod_list .product_list_menu ul {
+      margin: 0;
+      display: flex;
+      flex-wrap: wrap; }
+      .prod_list .product_list_menu ul li {
+        flex: 0 1 24%;
+        margin-right: 1%;
+        border-bottom: 1px dashed #ccc;
+        padding-bottom: 10px;
+        padding: 0 12px 10px;
+        background-color: #fff;
+        position: relative;
+        margin-bottom: 30px; }
+        .prod_list .product_list_menu ul li:after {
+          content: "";
+          position: absolute;
+          right: -5px;
+          top: 50%;
+          transform: translateY(-50%);
+          width: 1px;
+          height: 65%;
+          box-shadow: 0 5px 3px 2px #f9f9f9;
+          background-color: #f9f9f9;
+          border-radius: 50% 50% / 20px 20px; }
+        .prod_list .product_list_menu ul li h4 {
+          font-size: 17px;
+          font-weight: 700;
+          height: 4.2em; }
+
+.buy_now {
+  position: relative;
+  display: flex;
+  align-content: end;
+  justify-content: flex-end; }
+  .buy_now #mobile_view {
```

Comment:
Rules for ids are very specific you don;t need to add more specificity here.

Reply:
done